changelog/unreleased/issue-3490

```diff
@@ -0,0 +1,10 @@
+Enhancement: Support for specifying file size in read-data-subset
+
+To check a subset of repository files, restic used to support two ways to
+select a subset - A specific range of pack files, or random percentage of 
+pack files. I have added a third method to select pack files - By specifying 

```

Comment:
Please write `We have added ...` here to better match the usual changelog style.